ds3-autogen-c/src/main/resources/templates/request-templates/AmazonS3GetRequest.ftl

```diff
@@ -0,0 +1,29 @@
+<#-- ********************************** -->
+<#-- Generate "AmazonS3 Get Request" -->
+<#--   Input: Source object             -->
+<#-- ********************************** -->
+<#list getRequests() as requestEntry>
+    <#if (requestEntry.getClassification().toString() == "spectras3") && (requestEntry.getVerb().toString() == "GET")>
```

Comment:
Which 2 code blocks?

the amazon requests can take args void, bucket_name, or bucket_name and object_name

the spectra requests are either void or resource_id

(the method signatures are different, and the subsequent param checking)